doc/_includes/tutorials/common/samples/copy-samples.md

```diff
@@ -40,15 +46,40 @@ then select the checkbox "Remove samples from current project (move samples)"
 
 ### Locking Samples (ONLY FOR SHARING NOT MOVING)
 
-If you are sharing samples and you do not want them to be modifiable in the destination project, select the checkbox "Prevent modification of samples in target project (only when copying samples)"
+If you are sharing samples and you do not want them to be modifiable in the destination project, select the checkbox "
+Prevent modification of samples in target project (only when copying samples)"
 
 ![Share samples locks checkbox.]({{ site.baseurl }}/images/tutorials/common/samples/share-lock-checkbox.png)
 
 **NOTE: Both checkboxes cannot be selected at the same time.**
 
+### Metadata Restrictions
+
+![Share samples metadata restrictions.]({{ site.baseurl }}/images/tutorials/common/samples/share-metadata.png)
+
+Samples are shared/moved with all their corresponding metadata. This step allows you to review the fields that are
+included with the sample as well as review the metadata restriction level that will used on the destination project.
+
+If the metadata field does not exist in it will be set to the current level in the current project.
```

Comment:
Change `current level` to `current metadata restriction level`

Reply:
Updated in 0e8f9a2a60ad5a105bb12339b85daf89fe3908f1